tests/test_shardformer/test_model/_utils.py

```diff
@@ -3,12 +3,12 @@
 from colossalai.shardformer import ShardConfig, ShardFormer
 
 
-def build_model(model_fn):
+def build_model(model_fn, enable_flash_attention):
```

Comment:
This change will break other tests? Since other tests use this function as well.